supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorInitializationActor.scala

```diff
@@ -0,0 +1,74 @@
+package cromwell.backend.impl.htcondor
+
+import akka.actor.Props
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.impl.htcondor.HtCondorInitializationActor._
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import wdl4s.util.TryUtil
+import wdl4s.{Call, ThrowableWithErrors, WdlExpression}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+import scalaz.Scalaz._
+
+object HtCondorInitializationActor {
+  val FailOnStderrDefaultValue = true
+
+  def props(workflowDescriptor: BackendWorkflowDescriptor, calls: Seq[Call], configurationDescriptor: BackendConfigurationDescriptor): Props =
+    Props(new HtCondorInitializationActor(workflowDescriptor, calls, configurationDescriptor))
+}
+
+class HtCondorInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
+                                  override val calls: Seq[Call],
+                                  override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+  /**
+    * Abort all initializations.
+    */
+  override def abortInitialization: Future[WorkflowAbortResponse] = ???
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  override def validate: Future[Unit] = {
+    //TODO: Expressions need to be resolved. EvalFunctions is needed. TBD in the PR.
+    //TODO: refactor old code to use validation functions from Validation object in Backend.
+    //TODO: Workflow options may need to be validated for JES.
+    //TODO: Move specifics from Jes to JesBackendInitializationActor.
+    Future {
+      TryUtil.sequence(calls map {
```

Comment:
I can do it but that means to return ErrorOr[same type] in all validation functions.
If we follow the idea of just returning success and failure and not the resolved value it may work since I does not need to return the value.